projects/common/src/time/time-duration.ts

```diff
@@ -26,6 +29,70 @@ export class TimeDuration {
     return `PT${this.toMillis() / 1000}S`;
   }
 
+  public toIso8601DurationStringMilitaryTime(): string {
```

Comment:
if the goal of this is to convert a duration object into an Iso8601 duration string, it should be done using the unit and value in the constructor.

Reply:
Agreed. I'd also just update the existing method, that was just implemented with a simple approach due to laziness.